Similarity_Between_Sherlock_stories.py

```diff
@@ -0,0 +1,336 @@
+"""
+This program calculates the cosine similarity of all Sherlock Holmes books written
+by Sir. Arthur Connan Doyle and the first book in the series, A Study In Scarlet.
+It graphs the cosine similarity against time to investiage the evolution of
+Doyle's writing.
+
+
+Author: MJ-McMillen
+"""
+from __future__ import division
+import requests #used to get books from guttenberg
+import math
+#used to plot
+import numpy as np
+import matplotlib.pyplot as plt
+import wikipedia
+
+
+#A_Study_In_Scarlet = requests.get('http://www.gutenberg.org/ebooks/244.txt.utf-8').text
+A_Study_In_Scarlet= open('astudyinscarlet.txt.utf-8')
+A_Study_In_Scarlet_lines = A_Study_In_Scarlet.readlines()
+full_Study = A_Study_In_Scarlet_lines[63:4713]
```

Comment:
It seems like you manually found the line numbers of when the text starts and end. What we would have liked to see is using python to parse the text and determine where the text starts and ends.